doc/source/microsoft/step-zero-azure-autoscale.rst

```diff
@@ -305,7 +305,7 @@ If you prefer to use the Azure portal see the `Azure Kubernetes Service quicksta
    If you create the cluster using the Azure Portal you must enable RBAC.
    RBAC is enabled by default when using the command line tools.
 
-   If you use the Azure interactive shell the included version of Helm (2.10) has a known bug so the Secure Helm step in the :ref:`setup-helm` section of this guide will not work. Either skip this step, or download the latest version of Helm.
+   If you use the Azure interactive shell the included version of Helm (2.10) has a known bug so the Secure Helm step in the :ref:`setup-helm` section of this guide will not work. You will need to download the latest version of Helm as the Helm Chart requires version ``>= 2.11`` in order to handle pod priorities.
```

Comment:
@consideRatio Let me know what you think of the wording. It's the same in `step-zero-azure.rst` too 😄

Reply:
Oh if you have not used the Azure interactive shell, then I bet it may already be updated to a newer version by now. Could you look what version they use, and potentially remove this entirely if they have bumped it since this text was originally written?

Helm 2.10 is quite old by now so I'm surprised if they don't have a newer version available. I failed to google what version they have and only found "it comes preinstalled".

Reply:
I get the following error when I do `az aks install-cli` in the shell:
```
Connection error while attempting to download client ([Errno 13] Permission denied: '/usr/local/bin/kubectl')
```
This is probably why I never used it!

Reply:
Tagging @manics so we're not talking in multiple threads ✨

Reply:
IIRC you shouldn't need to install anything, the interactive shell can be used in your browser in https://portal.azure.com/

Reply:
🤦‍♀ Thank you @manics !

Seems like you get `helm` and `tiller` v 2.13.1 by default in the interactive shell now, so I'll delete the line!